### DISCUSSION TITLE
[A-L] (2023/24) Foglio 3 - Esercizio 2

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#149] [Esercizi - Bertini] [answered]
![Es 2](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/07c234ce-4afa-4b82-89bd-709194a27d38)

**Comment by Elia-Belli** [comment, marked as answer]
# Soluzione confermata in classe
Numeriamo le carte e definisco lo spazio elementare delle mie possibili mani $\Omega = \\{(w_1,..,w_{10})|w_i=1,..,40,\ w_i\ distinti\\}$
$|\Omega| =  \binom{40}{10}$
- **(A)** Mi assegno asso, due e tre di bastoni e pesco le restanti carte: $|A| = \binom{37}{7}$, $\mathbb{P}(A)= \binom{37}{7}/\binom{40}{10}$
- **(B)** Mi assegno asso, due e tre di bastoni e coppe, e pesco le restanti carte: $|B| = \binom{34}{4}$, $\mathbb{P}(B)= \binom{34}{4}/\binom{40}{10}$
- **(C)** Aggiornato, analogo a @CarloDaRomadev:
Considero $C$ come l'unione dei seguenti eventi disgiunti:
$n_3=\\{esattamente\ 3\ napoletane\\}\rightarrow |n_3|= \binom{4}{3}\binom{31}{1}$
$n_2=\\{esattamente\ 2\ napoletane\\}\rightarrow |n_2|= \binom{4}{2}\binom{34}{4}- 3\cdot |n_3|$ (perchè conterei le stesse napoletane triple 3 volte)
$n_1=\\{esattamente\ 1\ napoletana\\}\rightarrow |n_1|= \binom{4}{1}\binom{37}{7}- (2\cdot |n_2|+3\cdot |n_3|)$ (perchè conterei le stesse napoletane doppie 2 volte e le triple 3 volte)
$\mathbb{P}(C)= \mathbb{P}(almeno\ una\ napoletana)= \mathbb{P}(n_1)+\mathbb{P}(n_2)+\mathbb{P}(n_3)= \bigg(4\binom{37}{7}-\binom{4}{2}\binom{34}{4}+\binom{4}{3}\binom{31}{1}\bigg)/\binom{40}{10}$

**Comment by LeoCosta02** [comment]
@Elia-Belli I primi due punti li ho fatti uguali a te, il punto c però bisogna considerare anche il caso in cui ho due o 3 napoli perchè il testo dice almeno uno e i modi per farlo sono di più

**Comment by Elia-Belli** [comment]
> @Elia-Belli I primi due punti li ho fatti uguali a te, il punto c però bisogna considerare anche il caso in cui ho due o 3 napoli perchè il testo dice almeno uno e i modi per farlo sono di più

 Invece sono inclusi, quando dico che pesco dalle restanti carte non sto escludendo gli assi, due, tre degli altri semi: quindi se la mia mano è $1B,2B,3B$ posso ancora pescare $1S,2S,3S$ o addirittura una terza napoletana $1C,2C,3C$, dalle restanti 37 carte.

**Comment by daromacarlo** [comment]
@Elia-Belli nel punto 3 stai dicendo che per esempio 1B,2B,3B,1S,2S,3S,4B,5B,6B,7B e 1S,2S,3S,1B,2B,3B,4B,5B,6B,7B sono uguali o sbaglio?

**Comment by daromacarlo** [comment]
@Elia-Belli aspe' ho sbagliato a scrivere, intendevo un altra cosa, ho risolto.

**Comment by LeoCosta02** [comment]
![IMG_8912](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/128373356/25bb9ac7-f8ef-48c9-a401-603e084c515e)

**Comment by LeoCosta02** [comment]
Su un forum hanno risolto il punto 3 così

**Comment by daromacarlo** [comment]
# Soluzione confermata in classe
@LeoCosta02 a me il 3' usciva simile 
![Immagine 2023-10-21 195506](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/135441131/1abda750-0fa5-4517-886d-518640afe6c2)
però da come ho capito non serve farlo con il pricipio di inclusione esclusione dato che il binomiale elimina le ripetizioni posizionali

**Comment by LeoCosta02** [comment]
![IMG_8929](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/128373356/7ec5ae41-017a-46f9-96d8-9fac1481f679)

**Comment by LeoCosta02** [comment]
Non so se si capisce molto ma il ragionamento era partire dal binomiale 4(37 7 ) e togliere le ripetizioni in cui ho due napoli e tre napoli

**Comment by dahKidou** [comment]
> > @Elia-Belli I primi due punti li ho fatti uguali a te, il punto c però bisogna considerare anche il caso in cui ho due o 3 napoli perchè il testo dice almeno uno e i modi per farlo sono di più
> 
> Invece sono inclusi, quando dico che pesco dalle restanti carte non sto escludendo gli assi, due, tre degli altri semi: quindi se la mia mano è 1B,2B,3B posso ancora pescare 1S,2S,3S o addirittura una terza napoletana 1C,2C,3C, dalle restanti 37 carte.

Eh, però con questo ragionamento non tornano i primi due, perchè il testo delle prime due richieste esclude tutte le altre napoletane, no?

**Comment by Elia-Belli** [comment]
@dahKidou Nelle prime due non chiede "esattamente", quindi non esclude che te ne abbia altre.

**Comment by dahKidou** [comment]
> @dahKidou Nelle prime due non chiede "esattamente", quindi non esclude che te ne abbia altre.

Però la terza domanda specifica che la mano ne contenga ALMENO una. Nelle prime due era sottinteso e nella terza no? Non lo so, ho i miei dubbi.

**Comment by fraxb04** [comment]
> # Soluzione confermata in classe
> @LeoCosta02 a me il 3' usciva simile ![Immagine 2023-10-21 195506](https://private-user-images.githubusercontent.com/135441131/277125011-1abda750-0fa5-4517-886d-518640afe6c2.JPG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDE0MjY0NDcsIm5iZiI6MTcwMTQyNjE0NywicGF0aCI6Ii8xMzU0NDExMzEvMjc3MTI1MDExLTFhYmRhNzUwLTBmYTUtNDUxNy04ODZkLTUxODY0MGFmZTZjMi5KUEc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBSVdOSllBWDRDU1ZFSDUzQSUyRjIwMjMxMjAxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDIzMTIwMVQxMDIyMjdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hMDY3ODJlOTIwN2I3NTg5YjQxOWM2ZGQ0YmE0NmNiMTYwYWE2ZTkyNjcwNWJjY2U5OGM0YTBhODlkYzIyZWI4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.xLvOcrpjqdjqhQq8qFnOcWFDZrqujqmUNBSoEn5garQ) però da come ho capito non serve farlo con il pricipio di inclusione esclusione dato che il binomiale elimina le ripetizioni posizionali

potresti spiegarmi il tutto che non riesco proprio a capire :')

**Comment by daromacarlo** [comment]
@fraxb04 scusa la calligrafia, appena ho un attimo te lo riscrivo per bene e te lo spiego

**Comment by CiottoloMaggico** [comment]
![immagine](https://github.com/user-attachments/assets/56465b37-786c-48c4-aadc-ceb2fcad69c9)
